### PR TITLE
Update CONTRIBUTING.md

### LINKED ISSUE #144: Add the usage of `git lfs` in `CONTRIBUTING.md`
To download the large dataset in `pyrealm_build_data/`, the developer needs to run `git lfs pull`. Perhaps it's better to add this step in `CONTRIBUTING.md`. Running code testing without this step will lead to two errors related to the dataset.

### PR DESCRIPTION
**Pull request by tztsai**
## Fixes #144 

Add the instruction of using Git LFS in `CONTRIBUTING.md` to download the datasets, in order to pass two tests related to the dataset `inputs_data_24.25.nc`.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -53,6 +53,12 @@ Note that `pyrealm_build_data` is a source distribution only (`sdist`) component
 `pyrealm`, so is not included in binary distributions (`wheel`), such as those available
 from PyPi.
 
+### Fetching the datasets using Git LFS
+
+The datasets in `pyreal_build_data` are stored via Git LFS. After cloning the repository,
```
Comment (MarionBWeinzierl) [inline CONTRIBUTING.md:58]:
```suggestion
The datasets in `pyrealm_build_data` are stored via [Git LFS](https://git-lfs.com/). After cloning the repository,
```

**Review by MarionBWeinzierl — CHANGES_REQUESTED** [review]
That should probably be enough info. Please fix the typo and add a link to more info about Git LFS (you can simply commit my suggestions for that).

**Comment by MarionBWeinzierl** [comment]
There is now a formatting error flagged by the linting which needs to be resolved: 
image

**Comment by MarionBWeinzierl** [comment]
Now it has trailing whitespace (don't worry, happens to me all the time!):
image

**Comment by davidorme** [comment]
This shouldn't happen if `pre-commit` is installed? @TZTsai - have you got `pre-commit` set up on your machine? If you have then it shouldn't permit you to push commits that fail the `qa` action.

**Comment by tztsai** [comment]
> This shouldn't happen if `pre-commit` is installed? @TZTsai - have you got `pre-commit` set up on your machine? If you have then it shouldn't permit you to push commits that fail the `qa` action. That's actually described in `CONTRIBUTING.md` 😄

Ah sorry. I do have `pre-commit` installed, but I directly edited the file on GitHub.

**Comment by davidorme** [comment]
That explains it 😄

**Review by MarionBWeinzierl — APPROVED** [review]

**Pull request merged**